### PR TITLE
Update What's new page

### PR DESCRIPTION
**Pull request by yalgaer**
Please add what I missed and spelling errors.

+ I added the "What's New in UGENE" action to the Help menu.

### CODE REVIEW

**Review by smlnkcv — COMMENTED** [review]
Typo: Исправлены **ошибоки** при работе с файлами более 2Гб.

**Comment by yalgaer** [comment]
> Typo: Исправлены **ошибоки** при работе с файлами более 2Гб.

Fixed

File: src/ugeneui/html/version_news_en.html [hunk 1]
```diff
@@ -18,14 +18,34 @@
  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston,
  MA 02110-1301, USA.
 -->
-<h3>Changes in UGENE v. 39</h3>
+<h3>Changes in UGENE v. 40</h3>
+
+<h4>Alignment Editor</h4>
+<ul>
+    <li style="margin-bottom: 10px;"> Multi-row selection support.</li>
```
Comment (DmitriiSukhomlinov) [inline src/ugeneui/html/version_news_en.html]: `10px;"> Multi` - extra space after ">"

File: src/ugeneui/html/version_news_en.html [hunk 2]
```diff
@@ -18,14 +18,34 @@
...
+    <li style="margin-bottom: 10px;">Built Tree dialog now has all PhyML algorithms listed.</li>
```
Comment (DmitriiSukhomlinov) [inline src/ugeneui/html/version_news_en.html]: **The "Built Tree" dialog** now has all PhyML algorithms listed.

File: src/ugeneui/html/version_news_en.html (hunk 1 above)
Comment (yalgaer) [reply src/ugeneui/html/version_news_en.html]: Fixed.

File: src/ugeneui/html/version_news_en.html (hunk 2 above)
Comment (yalgaer) [reply src/ugeneui/html/version_news_en.html]: Fixed. Thank you.

File: src/ugeneui/html/version_news_en.html [hunk 3]
```diff
@@ -18,14 +18,34 @@
...
+    <li style="margin-bottom: 10px;">A new option to move selected sequences from one document to another.</li>
```
Comment (DmitriiSukhomlinov) [inline src/ugeneui/html/version_news_en.html:27]: A new option **of selected sequences moving**  from one document to another.

File: src/ugeneui/html/version_news_en.html [hunk 4]
```diff
@@ -18,14 +18,34 @@
...
+    <li style="margin-bottom: 10px;">Grouping mode: sorting by the size of a group and showing group sizes in the name list.</li>
```
Comment (DmitriiSukhomlinov) [inline src/ugeneui/html/version_news_en.html]: **The grouping mode**: sorting by the size of a group and showing group sizes in the name list.

**Review by smlnkcv — DISMISSED** [review]

File: src/ugeneui/html/version_news_en.html [hunk 5]
```diff
@@ -18,14 +18,34 @@
...
+</ul>
+
+<h4>Sanger Editor</h4>
+<ul>
+    <li style="margin-bottom: 10px;">A new visualization mode: show alternative mutations.</li>
+    <li style="margin-bottom: 10px;">Fixed issues with a vertical scrolling.</li>
```
Comment (DmitriiSukhomlinov) [inline src/ugeneui/html/version_news_en.html]: It sounds a bit weird, I'd rather replace it like:
`Vertical scrolling bugs have been fixed`

File: src/ugeneui/html/version_news_en.html (hunk 3 above)
Comment (yalgaer) [reply src/ugeneui/html/version_news_en.html:27]: Both variants "A new option to move" or "A new option of moving" are good. "of selected sequence moving" sounds very unnatural.
I prefer the first one, it is more widely used. Compare:  counts of results for 
* https://www.google.com/search?q=%22A+new+option+to+pay%22
vs
* https://www.google.com/search?q=%22A+new+option+of+paying%22

File: src/ugeneui/html/version_news_en.html (hunk 4 above)
Comment (yalgaer) [reply src/ugeneui/html/version_news_en.html]: Your suggestion is valid. Fixed. 
But my variant is also good and sounds more like a newspaper heading where articles usually omitted.

File: src/ugeneui/html/version_news_en.html (hunk 5 above)
Comment (yalgaer) [reply src/ugeneui/html/version_news_en.html]: Fixed

File: src/ugeneui/html/version_news_en.html [hunk 6]
```diff
@@ -18,14 +18,34 @@
...
+
+<h4>Sanger Editor</h4>
+<ul>
+    <li style="margin-bottom: 10px;">A new visualization mode: show alternative mutations.</li>
+    <li style="margin-bottom: 10px;">Vertical scrolling bugs have been fixed.</li>
+</ul>
+
+<h4>Sequence Viewer</h4>
+<ul>
+    <li style="margin-bottom: 10px;">Improved speed, memory usage and precision of sequence graphs.</li>
```
Comment (DmitriiSukhomlinov) [inline src/ugeneui/html/version_news_en.html:39]: `Speed, memory usage and precision of sequence graphs have been improved.`

File: src/ugeneui/html/version_news_en.html [hunk 7]
```diff
@@ -18,14 +18,34 @@
...
+    <li style="margin-bottom: 10px;">Fixed stability problems of auto-annotation routines.</li>
```
Comment (DmitriiSukhomlinov) [inline src/ugeneui/html/version_news_en.html:40]: `Stability problems of auto-annotation routines have been fixed.`

File: src/ugeneui/html/version_news_en.html [hunk 8]
```diff
@@ -18,14 +18,34 @@
...
+    <li style="margin-bottom: 10px;">Sequences with sizes above 2Gb now can be opened by UGENE again.</li>
```
Comment (DmitriiSukhomlinov) [inline src/ugeneui/html/version_news_en.html:41]: `Bugs, which appears while working with sequences larger when 2Gb, have been fixed.`

File: src/ugeneui/html/version_news_en.html [hunk 9]
```diff
@@ -18,14 +18,34 @@
...
+
+<h4>Other improvements</h4>
 <ul>
-    <li style="margin-bottom: 10px;"> Phylogenetic tree viewer now allows setting custom colors and fonts for subtree branches and labels.</li>
-    <li style="margin-bottom: 10px;"> The "Align to alignment" option got support for algorithm selection: ClustalO, MAFFT, Muscle, or UGENE.</li>
-    <li style="margin-bottom: 10px;"> Alignments with the RAW (undefined) alphabet can be converted to the DNA, RNA, or Amino alphabets.</li>
-    <li style="margin-bottom: 10px;"> Improved Unicode support for FASTA, MSF, and CLUSTALW formats.</li>
-    <li style="margin-bottom: 10px;"> Sequence Viewer got a new option to show codons or protein statistics for selected regions.</li>
-    <li style="margin-bottom: 10px;"> UGENE now speaks Turkish. Use menu Settings/Preferences to update the language.</li>
+    <li style="margin-bottom: 10px;">Improved CUDA build and error detection support.</li>
```
Comment (DmitriiSukhomlinov) [inline src/ugeneui/html/version_news_en.html:46]: `The "Smith-Waterman" algorithm for CUDA has been improved.`

File: src/ugeneui/html/version_news_en.html [hunk 10]
```diff
@@ -18,14 +18,34 @@
...
+    <li style="margin-bottom: 10px;">Unicode supported was added for Newick, Plain text, Diff, VCF, SNP file formats.</li>
```
Comment (DmitriiSukhomlinov) [inline src/ugeneui/html/version_news_en.html:47]: `The Unicode support has been added for Newick, Plain text, Diff, VCF, SNP file formats.`

File: src/ugeneui/html/version_news_en.html [hunk 11]
```diff
@@ -18,14 +18,34 @@
...
+    <li style="margin-bottom: 10px;">Random Sequence Generator memory footprint and generation speed improvements.</li>
```
Comment (DmitriiSukhomlinov) [inline src/ugeneui/html/version_news_en.html:48]: `The "Random Sequence Generator" memory footprint and generation speed have been improved.`

File: src/ugeneui/html/version_news_en.html (hunk 6 above)
Comment (yalgaer) [reply src/ugeneui/html/version_news_en.html:39]: These 2 styles are both valid. I use header style, you suggest a correct "detailed description" style.
There are plenty of examples in the internet when header style is used for "Whats new"
https://www.slidescore.com/news/a/000_changelog
https://k9mail.app/k-9/changelog_5.4-MAINT_branch.xml

File: src/ugeneui/html/version_news_en.html (hunk 7 above)
Comment (yalgaer) [reply src/ugeneui/html/version_news_en.html:40]: Done

File: src/ugeneui/html/version_news_en.html (hunk 6 above)
Comment (yalgaer) [reply src/ugeneui/html/version_news_en.html:39]: I accepted you variant to move forward

File: src/ugeneui/html/version_news_en.html (hunk 8 above)
Comment (yalgaer) [reply src/ugeneui/html/version_news_en.html:41]: Done

File: src/ugeneui/html/version_news_en.html (hunk 9 above)
Comment (yalgaer) [reply src/ugeneui/html/version_news_en.html:46]: Done

File: src/ugeneui/html/version_news_en.html (hunk 10 above)
Comment (yalgaer) [reply src/ugeneui/html/version_news_en.html:47]: Done

File: src/ugeneui/html/version_news_en.html (hunk 11 above)
Comment (yalgaer) [reply src/ugeneui/html/version_news_en.html:48]: Done

**Pull request merged**